media_driver/linux/common/ddi/media_libva_common.h

```diff
@@ -679,4 +679,42 @@ void* DdiMedia_GetContextFromVABufferID (PDDI_MEDIA_CONTEXT mediaCtx, VABufferID
 //!
 bool DdiMedia_DestroyBufFromVABufferID (PDDI_MEDIA_CONTEXT mediaCtx, VABufferID bufferID);
 
+//!
+//! \brief  Get  Gpu priority
+//!
+//! \param  [in] ctx
+//!     Pointer to VA driver context
+//! \param  [in] buffers
+//!     VA buffer ID
+//! \param  [in] numBuffers
+//!     Number of buffers
+//! \param  [out] priority_update
+//!     update priority
+//! \param  [out] priority
+//!     priority
+//! \return     int32_t
+//!
+int32_t DdiMedia_GetGpuPriority(
+    VADriverContextP     ctx,
+    VABufferID           *buffers,
+    int32_t              numBuffers,
+    bool                 *bPriorityUpdate,
```

Comment:
Please follow/align the naming rules of variables.

Reply:
Done.